examples/todo-list/src/controllers/todo-list-todo.controller.ts

```diff
@@ -13,33 +13,68 @@ export class TodoListTodoController {
     @repository(TodoListRepository) protected todoListRepo: TodoListRepository,
   ) {}
 
-  @post('/todo-lists/{id}/todos')
-  async create(@param.path.number('id') id: number, @requestBody() todo: Todo) {
+  @post('/todo-lists/{id}/todos', {
+    responses: {
+      '200': {
+        description: 'TodoList.Todo model instance',
+        content: {'application/json': {'x-ts-type': Todo}},
+      },
+    },
+  })
+  async create(
+    @param.path.number('id') id: number,
+    @requestBody() todo: Todo,
+  ): Promise<Todo> {
     return await this.todoListRepo.todos(id).create(todo);
   }
 
-  @get('/todo-lists/{id}/todos')
+  @get('/todo-lists/{id}/todos', {
+    responses: {
+      '200': {
+        description: "Array of Todo's belonging to TodoList",
+        content: {
+          'application/json': {
+            schema: {type: 'array', items: {'x-ts-type': Todo}},
+          },
+        },
+      },
+    },
+  })
   async find(
     @param.path.number('id') id: number,
     @param.query.string('filter') filter?: Filter,
-  ) {
+  ): Promise<Todo[]> {
     return await this.todoListRepo.todos(id).find(filter);
   }
 
-  @patch('/todo-lists/{id}/todos')
+  @patch('/todo-lists/{id}/todos', {
+    responses: {
+      '200': {
+        description: 'TodoList.Todo PATCH success count',
+        content: {'application/json': {'x-ts-type': Number}},
```

Comment:
Hmm, a response returning just a number is not very usual in REST/JSON world, is it?

I think it would be better to return an object with a `count` property instead, such response is easier to extend with additional properties while preserving backwards compatibility.

Having said that, such change is out of scope of this pull request.

Reply:
I agree it's unusual but it's what out Repository returns. I'll make a follow up issue for us to return an object instead.